docs/sources/source/nlp/ner.rst

```diff
@@ -46,7 +46,7 @@ Each line of the text.txt file contains text sequences, where words are separate
 You can use `this`_ to convert CoNLL-2003 dataset to the format required for training.
 
 
-.. _this: https://github.com/NVIDIA/NeMo/blob/master/scripts/convert_iob_format_to_token_classification_format.py
+.. _this: https://github.com/NVIDIA/NeMo/blob/master/token_classification/convert_iob_format_to_token_classification_format.py
```

Comment:
new file name?

Reply:
Done. Please check it to see if it is right.